crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -85,6 +85,20 @@ pub(super) fn place_comment<'a>(
             handle_leading_class_with_decorators_comment(comment, class_def)
         }
         AnyNodeRef::StmtImportFrom(import_from) => handle_import_from_comment(comment, import_from),
+        AnyNodeRef::StmtWhile(stmt_while) => {
+            handle_terminal_comment(comment, stmt_while.into(), locator)
+        }
```

Comment:
nit: I'm a bit biased towards `from` as I find it more readable although one could argue against it being verbose :)

```suggestion
        AnyNodeRef::StmtWhile(stmt_while) => {
            handle_terminal_comment(comment, TerminalExpression::from(stmt_while), locator)
        }
```